src/features/Authentication/login/login-container.jsx

```diff
@@ -16,12 +17,17 @@ function LoginContainer() {
     formState: { errors },
     reset,
   } = useForm({
+    mode: "all",
     resolver: yupResolver(schema),
   });
 
-  const onSubmitHandler = (data) => {
-    console.log({ data });
-    reset();
+  const onSubmitHandler = async (data) => {
+    try {
+      await login(data.email, data.password);
+      reset();
+    } catch (error) {
+      console.log(error);
+    }
   };
```

Comment:
How will the user know if there is an error, for example the email or password is wrong.
In the catch block where there is an error, firebase throws an error message to help you know what went wrong

What you can do is to set it to state and display it to the user


```jsx
const [errorMessage, setErrorMessage] = useState('')

try {
// -- login code

} catch(error){
setErrorMessage(error.message)
}

```